test/functional/smart_answers_controller_test.rb

```diff
@@ -116,7 +116,7 @@ def teardown
 
     should "display first question after starting" do
       get :show, params: { id: "smart-answers-controller-sample", started: "y" }
-      assert_select ".step.current [data-test=question]", /Do you like chocolate\?/
+      assert_select ".step.current .govuk-fieldset__legend", /Do you like chocolate\?/
```

Comment:
Do we know if the `.step.current` class + associated styling are still needed? I tried removing it and it looks ok, but not sure if I'm missing a situation where it's needed as it also looks ok on the live site without it.

Reply:
We should be using govuk-frontend for all the layout stuff so yes, this should definitely be looked at. I decided that this PR was big enough already without trying to take out govuk_frontend_toolkit at the same time so I'm going to address that in a followup PR.